src/components/svelte-react.ts

```diff
@@ -38,6 +38,7 @@ type Ref<T> =
     }
 export type ReactProps<Props> = Props & {
   ref?: Ref<Props>
+  slot?: string
```

Comment:
Does `ref?: Ref<Props>` not need a comma?

Reply:
It was `npm run format` which removed it

But also, nope :laughing: In Ts, valid separators are `\n`, `,` and `;` 

You can even mix/match as you see fit, if you're a monster.